package.json

```diff
@@ -54,6 +53,8 @@
     "eslint": "^6.8.0",
     "mocha": "10",
     "node-gyp": "^9.4.0",
+    "node-addon-api": "^7.0.0",
```

Comment:
You're right, currently this is needed for every users, I'v moved it to "dependencies". I think this can be move to dev deps again if we ship prebuilt binary to users one day, but not now.